requirements/main.txt

```diff
@@ -1,2 +1,2 @@
 pytest==7.1.2
-torch==1.13
+torch>=1.13
```

Comment:
we can't do this, we have to restrict to`1.13.*` (only give freedom for minor version number), otherwise we may have incapability issues once pytorch release `2.0`.